From f24baf240fce03b97901d7623ea838fc33d8773f Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Thu, 27 Jul 2023 18:17:32 -0400
Subject: [PATCH] yaffs2: v6.5 fixups

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/yaffs2/yaffs_vfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/yaffs2/yaffs_vfs.c b/fs/yaffs2/yaffs_vfs.c
index 66ce3efb6ad1..279be14981f5 100644
--- a/fs/yaffs2/yaffs_vfs.c
+++ b/fs/yaffs2/yaffs_vfs.c
@@ -844,7 +844,7 @@ static const struct file_operations yaffs_file_operations = {
 	.mmap = generic_file_mmap,
 	.flush = yaffs_file_flush,
 	.fsync = yaffs_sync_object,
-	.splice_read = generic_file_splice_read,
+	.splice_read = copy_splice_read,
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 16, 0)
 	.splice_write = iter_file_splice_write,
 #else
-- 
2.39.2

