From cb1d8a6453697d69b3de81c9d6f325271dc06cbd Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Fri, 11 Jul 2025 13:20:03 -0400
Subject: [PATCH 6/8] aufs: 6.16 fixups

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/aufs/vfsub.c | 4 ++--
 fs/namespace.c  | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/fs/aufs/vfsub.c b/fs/aufs/vfsub.c
index e1fe4ac876e5..5e05229ef329 100644
--- a/fs/aufs/vfsub.c
+++ b/fs/aufs/vfsub.c
@@ -219,7 +219,7 @@ struct dentry *vfsub_lookup_one_len_unlocked(const char *name,
 {
 	struct path path;
 
-	path.dentry = lookup_one_len_unlocked(name, ppath->dentry, len);
+	path.dentry = lookup_noperm_unlocked(name, ppath->dentry, len);
 	if (IS_ERR(path.dentry))
 		goto out;
 	if (d_is_positive(path.dentry)) {
@@ -240,7 +240,7 @@ struct dentry *vfsub_lookup_one_len(const char *name, struct path *ppath,
 	/* VFS checks it too, but by WARN_ON_ONCE() */
 	IMustLock(d_inode(ppath->dentry));
 
-	path.dentry = lookup_one_len(name, ppath->dentry, len);
+	path.dentry = lookup_noperm(name, ppath->dentry, len);
 	if (IS_ERR(path.dentry))
 		goto out;
 	if (d_is_positive(path.dentry)) {
diff --git a/fs/namespace.c b/fs/namespace.c
index e1b1390762eb..1c172584695e 100644
--- a/fs/namespace.c
+++ b/fs/namespace.c
@@ -2464,7 +2464,6 @@ static bool __has_locked_children(struct mount *mnt, struct dentry *dentry)
 	}
 	return false;
 }
-EXPORT_SYMBOL_GPL(drop_collected_mounts);
 
 bool has_locked_children(struct mount *mnt, struct dentry *dentry)
 {
-- 
2.39.2

