From 05e1cb770859bcf5ca080c1482a84dc23fbb294b Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Mon, 14 Jul 2025 12:53:42 -0400
Subject: [PATCH 7/8] aufs: update lookup_noperm* to v6.16 signatures

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/aufs/vfsub.c | 14 ++++++++++++--
 1 file changed, 12 insertions(+), 2 deletions(-)

diff --git a/fs/aufs/vfsub.c b/fs/aufs/vfsub.c
index 5e05229ef329..483548be86b4 100644
--- a/fs/aufs/vfsub.c
+++ b/fs/aufs/vfsub.c
@@ -218,8 +218,13 @@ struct dentry *vfsub_lookup_one_len_unlocked(const char *name,
 					     struct path *ppath, int len)
 {
 	struct path path;
+	struct qstr this = {
+		.name = name,
+		.len = len,
+		.hash = 0	/* will be set by lookup function if needed */
+	};
 
-	path.dentry = lookup_noperm_unlocked(name, ppath->dentry, len);
+	path.dentry = lookup_noperm_unlocked(&this, ppath->dentry);
 	if (IS_ERR(path.dentry))
 		goto out;
 	if (d_is_positive(path.dentry)) {
@@ -239,8 +244,13 @@ struct dentry *vfsub_lookup_one_len(const char *name, struct path *ppath,
 
 	/* VFS checks it too, but by WARN_ON_ONCE() */
 	IMustLock(d_inode(ppath->dentry));
+	struct qstr this = {
+		.name = name,
+		.len = len,
+		.hash = 0	/* will be set by lookup function if needed */
+	};
 
-	path.dentry = lookup_noperm(name, ppath->dentry, len);
+	path.dentry = lookup_noperm(&this, ppath->dentry);
 	if (IS_ERR(path.dentry))
 		goto out;
 	if (d_is_positive(path.dentry)) {
-- 
2.39.2

