From 1785f762bbbbbef09773e0af95d8d58acb9abfd3 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Wed, 14 Dec 2022 12:34:00 -0500
Subject: [PATCH 1/5] aufs6: kbuild

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/Kconfig  | 1 +
 fs/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/fs/Kconfig b/fs/Kconfig
index 47c5dcd80e56..cbd949b073cb 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -320,6 +320,7 @@ source "fs/sysv/Kconfig"
 source "fs/ufs/Kconfig"
 source "fs/erofs/Kconfig"
 source "fs/vboxsf/Kconfig"
+source "fs/aufs/Kconfig"
 
 endif # MISC_FILESYSTEMS
 
diff --git a/fs/Makefile b/fs/Makefile
index 6b32a53b25f4..387dd0c6afce 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -138,3 +138,4 @@ obj-$(CONFIG_EROFS_FS)		+= erofs/
 obj-$(CONFIG_VBOXSF_FS)		+= vboxsf/
 obj-$(CONFIG_ZONEFS_FS)		+= zonefs/
 obj-$(CONFIG_YAFFS_FS)		+= yaffs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
-- 
2.19.1

