From 4e0173f1c63ba3e1c2503af8b563c4c1940aa1b1 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@gmail.com>
Date: Fri, 19 Mar 2021 12:33:42 -0400
Subject: [PATCH 4/5] aufs5: aufs-kbuild

Signed-off-by: Bruce Ashfield <bruce.ashfield@gmail.com>
---
 fs/Kconfig  | 1 +
 fs/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/fs/Kconfig b/fs/Kconfig
index a3bb10cff321..bc30d910ab00 100644
--- a/fs/Kconfig
+++ b/fs/Kconfig
@@ -289,6 +289,7 @@ source "fs/sysv/Kconfig"
 source "fs/ufs/Kconfig"
 source "fs/erofs/Kconfig"
 source "fs/vboxsf/Kconfig"
+source "fs/aufs/Kconfig"
 
 endif # MISC_FILESYSTEMS
 
diff --git a/fs/Makefile b/fs/Makefile
index 14b056ece5fe..b9df02256367 100644
--- a/fs/Makefile
+++ b/fs/Makefile
@@ -136,3 +136,4 @@ obj-$(CONFIG_EROFS_FS)		+= erofs/
 obj-$(CONFIG_VBOXSF_FS)		+= vboxsf/
 obj-$(CONFIG_ZONEFS_FS)		+= zonefs/
 obj-$(CONFIG_YAFFS_FS)		+= yaffs2/
+obj-$(CONFIG_AUFS_FS)           += aufs/
-- 
2.19.1

