From 0d4a6d2bd9e209a4e1381df1c31d6c21135c5b3c Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Fri, 8 Jul 2011 20:25:16 +0200
Subject: [PATCH 184/184] Add localversion for -RT release

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
---
 localversion-rt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 localversion-rt

diff --git a/localversion-rt b/localversion-rt
new file mode 100644
index 000000000000..18777ec0c27d
--- /dev/null
+++ b/localversion-rt
@@ -0,0 +1 @@
+-rt15
-- 
2.34.1

